fluentd/configs.d/openshift/es-ops-copy-config.conf

```diff
@@ -11,6 +11,9 @@
       client_cert "#{ENV['OPS_COPY_CLIENT_CERT']}"
       ca_file "#{ENV['OPS_COPY_CA']}"
 
+      type_name ${(record && record.delete('es_type')) || 'fluentd'}
```

Comment:
I think we can move to a static type in the `match`, the additional benefit of that is  - we won't have the extra field `es_type` in ES.